impeller/renderer/backend/vulkan/swapchain/ahb/ahb_texture_pool_vk.h

```diff
@@ -98,17 +96,15 @@ class AHBTexturePoolVK {
 
  private:
   struct PoolEntry {
-    TimePoint last_access_time;
```

Comment:
I suppose the pool entry no longer needs to be a struct. This is still fine though.